### DISCUSSION TITLE
ATS and ETS2 concurrently

### DISCUSSION THREAD

**Discussion by mklengel** [LsHallo/ets2-dedicated-convoy-server#17] [Q&A]
I managed to run both (ATS and ETS2) servers simultanously in my Docker environment.
I thought different ports and that's it.

Problem is both servers (ATS and ETS2) are shown in ATS and also in ETS2.
What has to be done to show only the ATS server in ATS (same for ETS2 of course)?

**Comment by LsHallo** [comment]
Wow you seem to have the opposite problem of everybody else. They all can't see their server in game. I will try this myself and see what happens. Does not sound like what should happen. 

Can you verify that you assigned the correct login token to each server?

Maybe it's also worth creating a topic in the SCS forum. I can't imagine that it should even be possible to see ATS servers in ETS and they might want to look into this.

**Comment by mklengel** [comment]
Your answer points me to something... I have not configured login tokens. I'll have a look.

**Reply by LsHallo** [reply]
Wait are you using the search ID to search for your server?
That might explain the cross game availability. Although it still should not appear.

**Reply by mklengel** [reply]
I now have configured the tokens and the games (ETS and ATS) are working.
Two things are annoying.

1. Konvoi search: searching with the start of the lobby name (e.g. "Taunus") as the user who owns the server shows ATS and ETS2 server instead of the expected single ETS2.
2. Konvoi search: other users can't find this server with the lobby name, only via the Session ID.

Any idea whats wrong?
